app/controllers/application_pets_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ApplicationPetsController < ApplicationController
+  
+  def new
+  end
+
+  def create
+    @application_pet = ApplicationPet.new(application_pets_params)
+    @application_pet.save
+    redirect_to "/applications/#{@application_pet.application_id}"
+  end
```

Comment:
New and create actions are great places to make use of flash messages! What would happen if this action if the application_pet record didn't save?